pkg/runtime/k0s/runtime.go

```diff
@@ -38,48 +41,60 @@ type Runtime struct {
 	RegConfig *registry.Config
 }
 
+var ForceDelete bool
```

Comment:
I have not see any assignment to this variable. Please double check it.

Reply:
Using this in Line45